homeassistant/components/jewish_calendar/service.py

```diff
@@ -17,29 +18,56 @@
 )
 from homeassistant.helpers import config_validation as cv
 from homeassistant.helpers.selector import LanguageSelector, LanguageSelectorConfig
+from homeassistant.helpers.sun import get_astral_event_date
+from homeassistant.util import dt as dt_util
 
-from .const import ATTR_DATE, ATTR_NUSACH, DOMAIN, SERVICE_COUNT_OMER
+from .const import AFTER_SUNSET, ATTR_DATE, ATTR_NUSACH, DOMAIN, SERVICE_COUNT_OMER
 
+_LOGGER = logging.getLogger(__name__)
 SUPPORTED_LANGUAGES = {"en": "english", "fr": "french", "he": "hebrew"}
 OMER_SCHEMA = vol.Schema(
     {
-        vol.Required(ATTR_DATE, default=datetime.date.today): cv.date,
+        vol.Optional(ATTR_DATE): cv.date,
+        vol.Optional(AFTER_SUNSET, default=False): cv.boolean,
```

Comment:
I'm not sure I'd use False as the default, since the count is usually done after sunset... Usually people relate to a given day as to what will be counted after sunset.